packages/eas-cli/src/run/android/aapt.ts

```diff
@@ -0,0 +1,52 @@
+import spawnAsync, { SpawnResult } from '@expo/spawn-async';
+import glob from 'fast-glob';
+
+import Log from '../../log';
+import { getAndroidSdkRootAsync } from './sdk';
+
+async function aaptAsync(...options: string[]): Promise<SpawnResult> {
+  try {
+    return await spawnAsync(await getAaptExecutableAsync(), options);
+  } catch (error: any) {
+    const errorMessage = (error.stderr || error.stdout || error.message).trim();
+    error.message = errorMessage;
+    throw error;
+  }
+}
+
+export async function getAaptExecutableAsync(): Promise<string> {
+  const sdkRoot = await getAndroidSdkRootAsync();
+  if (!sdkRoot) {
+    Log.debug('Failed to resolve the Android SDK path, falling back to global aapt executable');
+    return 'aapt';
+  }
+  const aaptPaths = await glob(`${sdkRoot}/build-tools/**/aapt`);
+
+  if (aaptPaths.length === 0) {
+    throw new Error('Failed to resolve the Android aapt path');
+  }
+
+  return aaptPaths.sort().reverse()[0];
```

Comment:
This is more straightforward:
```suggestion
  const sorted = aaptPaths.sort();
  return sorted[sorted.length - 1];
```